Updates for Cypress v.10 CT testing (#503)
cypress-io/github-action: README.md

> ### Component tests

Reviewer: @robertguss We need to add 1 or 2 examples for v10 that have component testing running in this repo so that the `component: true` attribute is used and tested.

Author: @kevinold I have added the CRA with Chakra UI project and created an example-component-test.yml workflow for it.

Author: @kevinold I commented out the component test workflow for now since it depends upon code and updates in this PR. Once we merge this PR, we will need to uncomment it to get it working.

Reviewer: @robertguss Not sure what you mean, as we want those to be working as this PR is merged.  Let me review and see if I can understand, but we might need to sync on it.

Reviewer: @robertguss Testing this to see if it allows the CT example to run https://github.com/cypress-io/github-action/pull/503/commits/3e9a6baf78587adf47bc55ac99382c36b2233989

Reviewer: @robertguss It worked! https://github.com/cypress-io/github-action/runs/5208533374?check_suite_focus=true

Author: @kevinold awesome! I could have sworn I tried this before, but glad to know it works ;)

Author: yeah I tried it in this commit, but it didn't seem to work: https://github.com/cypress-io/github-action/pull/503/commits/4feb2488c38020bb1c472043bdfc3892719b0f3e

> You can run [Cypress component tests](https://on.cypress.io/component-testing) after running E2E tests by adding `component: true`:

Reviewer: @robertguss Updated this to clarify it would be a separate job
```suggestion
You can run [Cypress component tests](https://on.cypress.io/component-testing) in a job separate from E2E tests by adding `component: true`:
```
